src/buildercore/trop.py

```diff
@@ -200,15 +219,17 @@ def rdsinstance(context, template):
         'DBSubnetGroupName': Ref(rsn),
         'DBInstanceClass': lu('project.aws.rds.type'),
         'Engine': lu('project.aws.rds.engine'),
+        # something is converting this value to an int from a float :(
```

Comment:
yeah, I just updated the comment from my memory of the problem. it's why the engine version is quoted `'9.4'` in the elife.yaml projects file. I'll remove if it's not an issue